config.md

```diff
@@ -265,6 +265,19 @@ Annotations are key-value maps.
 }
 ```
 
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
imagePath needs to be absolute since it will not form part of the bundle, so the runtime needs to know exactly where on the host to find it.

I've updated the PR with further details on imagePath as suggested.